README.md

```diff
@@ -97,15 +110,190 @@ A quick look at the top-level files and directories you'll see in a Gatsby proje
+}
+
+export default Button
+```
+
+Now, our Button component is powered by Store UI. However, if you try to use this on your app you will see that the button is lacking styles. To add styles, we will use CSS modules because it allow us to target data attributes. On your terminal, type:
+```sh
+touch src/components/ui/Button/Button.module.css
+```
+
+Now, on `Button.module.css`:
+```css
+[data-store-button]: {
+  @apply p-0 bg-primary-100;
+}
+```
+
+The `@apply` directive exists because we are using [Tailwind css](https://tailwindcss.com/). To learn more about tailwind, [see their docs](https://tailwindcss.com/docs). To know more about our best practices on using tailwid, see the `Styling Components` section.
+This `data-store-button` is a css data attribute selector. To know which selectors are available, check [store-ui docs](https://storeui.netlify.app/)
+
+Now, open `Button.tsx` and import this CSS with:
+```tsx
+import React from 'react'
+import { Button as UIButton } from '@vtex/store-ui'
+import type { ButtonProps } from '@vtex/store-ui'
+
+import './Button.module.css'
+
+interface Props extends ButtonProps {}
+
+function Button (props: Props) {
+  return <UIButton {...props} />
+}
+
+export default Button
+```
+
+For most components you would stop here. However, buttons can have different variants. For instance, suppose you want to have a button component with primary and muted variants. To add variants to the component, you can, on `Button.tsx`
+```tsx
+import React from 'react'
+import { Button as UIButton } from '@vtex/store-ui'
+import type { ButtonProps } from '@vtex/store-ui'
+
+import './Button.module.css'
+
+interface Props extends ButtonProps {
+  variant: 'muted' | 'primary'
+}
+
+function Button ({variant, ...props}: Props) {
+  return <UIButton className={variant} {...props} />
+}
+
+export default Button
+```
+
+and than, on `Button.module.css`:
+```css
+.primary [data-store-button]: {
+  @apply p-0 bg-primary-100;
+}
+
+.muted [data-store-button]: {
+  @apply p-0 bg-muted-100;
+}
+```
+
+Now we have a styled Button component that accepts different variants!! 🎉
+
+The aforementioned guide works well for UI components. However, sections are more complex and usually don't have variants, since they usually serve a single responsibility in the page. For sections, you can use tailwind natively like:
+
+```tsx
+// components/sections/MySection/MySection.tsx
+...
+function MySection {
+  return (
+    <>
+      <div className="p-2 flex items-center">
+        <p className="h-4 p-5 mx-2">Hello World</p>
+        <Button variant="muted">
+      </div>
+    </>
+  )
+}
+...
+```
+
+## 🖊️ Styling Components
+This starter uses [Tailwind CSS](https://tailwindcss.com/) for styling. If you want, you can remove it and use other solutions. Both Gatsby and Store UI support many different CSS frameworks, like [emotion](https://emotion.sh/docs/introduction), [stitches](https://stitches.dev/docs/introduction) or even no CSS framework at all. 
```

Comment:
Is there something like a list of which CSS frameworks Gatsby supports?

Reply:
Nice! Just added the framework list link